### PR TITLE
Todo App

### PR DESCRIPTION
**Pull request by ZawadiSumba66**
lists added to a project each witha description, title and due date.
Each item in the list can be deleted and the a particular project too

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @ZawadiSumba66 👋

**Nice project, Well detailed readme and good use of class contructors. 👌**

However, let's make a couple of changes.

- [ ] Please refer to comments on the code.

- [ ] Kindly include an Edit functionality on todo items. A user should be able to edit a todo item. [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/ui_requirement.png)

- [ ] Kindly include a priority field to todo items. [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/todo_requirement.png)

- Please refer to other comments on the code.

_**To highlight**_
- Remove comments in the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/all.js [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+  submit.addEventListener('click', (e) => {
+    e.preventDefault();
+    project.createProject(name.value);
+    const arr = project.getProjectsArray();
+    addProjects(arr, arr.length - 1, projectDiv, todoDiv);
+    loadAllTodos(arr.length - 1, todoDiv);
+    form.reset();
+    // form.classList.add('hidden');
+    const todoForm = todoF;
+    todoForm.classList.add('hidden');
```
Comment (mikenath223) [inline src/all.js]: - [ ] Kindly remove these comments on the code.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import project from './project';
+import todo from './task';
+import page from './all';
+import storageModule from './storage';
+// import './reset.css';
```
Comment (mikenath223) [inline src/index.js]: - [ ] Please remove this commented code also.

**Review by bellom — APPROVED** [review]
## Hi @ZawadiSumba66  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**